index.js

```diff
@@ -0,0 +1,32 @@
+const menuBtn = document.querySelector('.menu-btn');
+const headerNavMobile = document.querySelector('.header-nav-mobile');
+const cancelCross = document.querySelector('.cancel-cross');
+const linkPortfolio = document.querySelector('.menu-link');
+const linkAbout = document.querySelector('.menu-link');
+const linkContact = document.querySelector('.menu-link');
+
+menuBtn.addEventListener('click', () => {
+    menuBtn.classList.add('hidden')
+    headerNavMobile.classList.remove('hidden')
+});
+
+cancelCross.addEventListener('click', () => {
+    menuBtn.classList.remove('hidden')
+    headerNavMobile.classList.add('hidden')
+});
+
+linkPortfolio.addEventListener('click', () => {
+    headerNavMobile.classList.add('hidden');
+    menuBtn.classList.remove('hidden');
+  });
+  
+  linkAbout.addEventListener('click', () => {
+    headerNavMobile.classList.add('hidden');
+    menuBtn.classList.remove('hidden');
+  });
+  
+  linkContact.addEventListener('click', () => {
+    headerNavMobile.classList.add('hidden');
+    menuBtn.classList.remove('hidden');
+  });
```

Comment:
- [ ] Your mobile menu is great, but by using the `querySelector()` tag you are only getting the first item of that class in the HTML, and unfortunately the 'click' events are not being applied to the other 2 mobile menu items. I suggest you use the `querySelectorAll()` method and it is not necessary to add the same event listener to multiple elements with the same class name, this way you will not repeat code and follow the best practices of DRY code. Kindly fix that so all the mobile menu links close the menu when they are clicked.